Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/Visuals/Configuration/TranslationHandlesConfiguration.cs

```diff
@@ -0,0 +1,123 @@
+{
+    /// <summary>
+    /// Configuration for <see cref="TranslationHandles"/> used in <see cref="BoundsControl"/>
+    /// This class provides all data members needed to create translation handles for <see cref="BoundsControl"/>
+    /// </summary>
+    [CreateAssetMenu(fileName = "TranslationHandlesConfiguration", menuName = "Mixed Reality Toolkit/Experimental/Bounds Control/Translation Handles Configuration")]
+    public class TranslationHandlesConfiguration : HandlesBaseConfiguration
+    {
+
+        [SerializeField]
+        [Tooltip("Determines the type of collider that will surround the translation handle prefab.")]
+        private HandlePrefabCollider translationHandlePrefabColliderType = HandlePrefabCollider.Box;
+
+        /// <summary>
+        /// Determines the type of collider that will surround the translation handle prefab.
+        /// </summary>
+        public HandlePrefabCollider TranslationHandlePrefabColliderType
+        {
+            get
+            {
+                return translationHandlePrefabColliderType;
+            }
+            set
+            {
+                if (translationHandlePrefabColliderType != value)
+                {
+                    translationHandlePrefabColliderType = value;
+                    colliderTypeChanged.Invoke();
+                }
+            }
+        }
+
+        [SerializeField]
+        [Tooltip("Check to show translation handles for the X axis")]
+        private bool showTranslationHandleForX = true;
+
+        /// <summary>
+        /// Check to show translation handles for the X axis
+        /// </summary>
+        public bool ShowTranslationHandleForX
+        {
+            get
+            {
+                return showTranslationHandleForX;
+            }
+            set
+            {
+                if (showTranslationHandleForX != value)
+                {
+                    showTranslationHandleForX = value;
+                    handlesChanged.Invoke(HandlesChangedEventType.Visibility);
+                }
+            }
+        }
+
+        [SerializeField]
+        [Tooltip("Check to show translation handles for the Y axis")]
+        private bool showTranslationHandleForY = true;
+
+        /// <summary>
+        /// Check to show translation handles for the Y axis
+        /// </summary>
+        public bool ShowTranslationHandleForY
+        {
+            get
+            {
+                return showTranslationHandleForY;
+            }
+            set
+            {
+                if (showTranslationHandleForY != value)
+                {
+                    showTranslationHandleForY = value;
+                    handlesChanged.Invoke(HandlesChangedEventType.Visibility);
+                }
+            }
+        }
+
+        [SerializeField]
+        [Tooltip("Check to show translation handles for the Z axis")]
+        private bool showTranslationHandleForZ = true;
```

Comment:
i'm wondering if we should expose this as flags as well - i can take care of doing the same for rotation handles - what do you think?

Reply:
Will do. We should make sure to do it in the same way, for consistency.

Reply:
Going to leave this to a separate PR for now.